### ISSUE TITLE
How to include images?

### ISSUE THREAD

**Issue by danieljfarrell** [skriptum/diatypst#1]
Hello,

Thank you for your very nice template!

I am trying to include an image, but I get an errors

```
Array is empty

Array index out of bounds (index: 1, len: 0)
```

when I do that?

I literally just added a #image("myimage") to a section.

Here is the full document

```
#import "@preview/diatypst:0.1.0": *

#show: slides.with(
  title: "Title", // Required
  subtitle: "Subtitle",
  date: "29.08.2024",
  authors: ("Me"),

  // Optional Styling
  ratio: 16/9,
  layout: "medium",
  title-color: aqua.darken(50%),
  footer: true,
  counter: true,
)

#outline()

= Section

== Sub-section

Remove this text to fix the issue.

#image("myimage.jpeg")
```

**Comment by skriptum** [comment]
Hi, 
thanks for the heads up and sorry for the delay. It seems to be a problem with the counter. As a short term fix, you can just include a slide afterwards (with a new Heading) and at least for me that solves the problem. I'll try to fix the problem

**Comment by skriptum** [comment]
Should be closed with the new version 0.2
